app/src/pages/OnDeviceDisplay/ProtocolSetup/index.tsx

```diff
@@ -174,13 +174,15 @@ export function ProtocolSetupStep({
             {subDetail}
           </StyledText>
         </Flex>
-        <Icon
-          marginLeft={SPACING.spacing8}
-          name="more"
-          size="3rem"
-          // Required to prevent inconsistent component height.
-          style={{ backgroundColor: disabled ? 'transparent' : 'initial' }}
-        />
+        {!disabled && (
+          <Icon
+            marginLeft={SPACING.spacing8}
+            name="more"
+            size="3rem"
+            // Required to prevent inconsistent component height.
+            style={{ backgroundColor: disabled ? 'transparent' : 'initial' }}
+          />
+        )}
```

Comment:
this totally works! You can also add a ternary which is more common in our repo i think

```suggestion
        {!disabled ? (
          <Icon
            marginLeft={SPACING.spacing8}
            name="more"
            size="3rem"
            // Required to prevent inconsistent component height.
            style={{ backgroundColor: disabled ? 'transparent' : 'initial' }}
          />
        ) : null}
```

Reply:
Great tip, I'll make that change